xml/admin_backup.xml

```diff
@@ -0,0 +1,394 @@
+    healthy at the time of backup.
+   </para>
+   <para>
+    This document does not describe recovery from a multi node cluster failure.
+   </para>
+   <para>
+    The scenario described in this document does not cover backup for persistent
+    data. It is supposed to allow restoring cluster functionality after node
+    failure. Please apply best practices to ensure proper retention of
+    persistent data.
+   </para>
+   <para>
+    Depending on the type of failure scenario, workloads are likely to be
+    recovered in Kubernetes but must be expected to be restarted in many
+    scenarios.
+   </para>
+   <para>
+    Always make sure your cluster has sufficient redundant capacity to
+    temporarily shift workloads to other workers during upgrades or failures.
+   </para>
+   <para>
+    The described approach is not suited for incremental backups.
+   </para>
+  </important>
+ </sect1>
+
+ <sect1 xml:id="sec.admin.backup">
+ <title>Backup Scenarios</title>
+ <note>
+  <title>Recovering Failed Nodes</title>
+  <para>
+   In some failure scenarios (network disconnect, simple hardware failure) nodes
+   can be recovered without additional action. Simply bringing the machines back
+   online will suffice to return them to the cluster.
+  </para>
+  <para>
+   In case of severe hardware failure (loss of hard drive, data corruption),
+   generally speaking, a replacement is advised and nodes should be replaced
+   with fresh (virtual) machines.
+  </para>
+ </note>
+
+  <sect2 xml:id="sec.admin.backup.admin">
+   <title>Admin Node Failure</title>
+   <para>
+    We assume:
+   </para>
+
+    <itemizedlist>
+     <listitem>
+      <para>
+       The &admin_node; has failed and is unrecoverable
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       The rest of the cluster is unaffected
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       The IP address / hostname for the replacement node will be identical
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       A backup of the &admin_node; has been taken from a healthy cluster state
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       The backup procedure is performed regularly
+      </para>
+     </listitem>
+    </itemizedlist>
+
+   <para>
+    The <xref linkend="sec.admin.backup.admin.restore"/> will involve deploying
+    a new &admin_node; machine and restoring the previous data onto it.
+    &productname; will then resume management of the cluster. Existing workloads
+    should not be affected.
+   </para>
+   <procedure>
+    <title>Backup procedure</title>
+    <step>
+     <para>
+      SSH into the &admin_node;.
+     </para>
+    </step>
+    <step>
+     <para>
+      Change to the <filename>/var/tmp</filename> directory and package
+      important data into a tar file archive:
+     </para>
+<screen>&prompt.root;<command>
+tar czf <replaceable>backup-admin-node-$( date +'%Y-%m-%d' ).tgz</replaceable> -C / \
```

Comment:
produces correct "backup-admin-node-2019-02-26.tgz" filename but personally I would remove spaces close to the `date` command so it would look like this:

`tar czf <replaceable>backup-admin-node-$(date +'%Y-%m-%d').tgz</replaceable> -C / \`